web/app/css/styles.css

```diff
@@ -5,11 +5,10 @@
   --buttonblue: #4076c4;
   --messageblue: #2F80ED;
   --darkblue: #071E3C;
+  --royalblue: #2D9CDB;
+  --pictonblue: #56CCF2;
   --warmgrey: #f6f6f6;
   --coldgrey: #c9c9c9;
-  --latency-p99: var(--messageblue);
```

Comment:
I think we should leave the `--latency-pXX` vars defined since we have multiple places that will want to be referencing the same color (for example the latency p50 summary and the latency p50 chart line should be the same color, and I think it's fine for that to be named `--latency-p50`), but the base variables can reference the new named blues:

```
  --curiousblue: #2D9CDB;
  --royalblue: #2F80ED;
  --pictonblue: #56CCF2;
  --latency-p99: var(--royalblue);
  --latency-p95: var(--curiousblue);
  --latency-p50: var(--pictonblue);
```

Then in line-graph.css we can use `fill: var(--pictonblue);` in line 26 (as you've done!) but leave lines 13-22 to reference the original --latency-pXX variables.